test_requirements.txt

```diff
@@ -0,0 +1,7 @@
+-r requirements.txt
+
+coveralls
+pep8
+git+https://github.com/pmitros/django-pyfs.git@514607d78535fd80bfd23184cd292ee5799b500d#egg=djpyfs
```

Comment:
I believe that we should be using the edx branch of this now as this is in edx-platform:
https://github.com/edx/edx-platform/blob/master/requirements/edx/github.txt#L58

@doctoryes can you confirm this?